styles/variables-silicon.less

```diff
@@ -213,19 +213,28 @@
 // Item
 // ---------------------------------------
 @agate-item-border-width:     0 0 3px 0;
+@agate-item-centered-inline-max-width: 480px;
 @agate-item-font-size:        33px;
 @agate-item-font-weight:      normal;
 @agate-item-height:           102px;
-@agate-item-height-large:     120px;
 @agate-item-side-outset:      30px;
 @agate-item-sub-text-size:    24px;
+@agate-item-inline-content-max-width: 639px;
```

Comment:
Would you please align this with the inline one?

Reply:
Actually this line will be removed from Silicon since I see there is a bug in storybook with Item inline and long labelAfter. The width of Item will grow on focus